db/seeds.rb

```diff
@@ -192,22 +214,18 @@
   city_town: 'Nonsense'
 )
 
-# create 10 pets
-
-10.times  do
+10.times do
   Pet.create!(
-    organization_id: 1,
+    organization: [@organization_one, @organization_two].sample,
```

Comment:
What are your thoughts on sampling from all Orgs? (less code to update in the future)

```suggestion
    organization: Organization.all.sample,
```